easybuild/easyconfigs/n/nifti2dicom/nifti2dicom-0.4.11-no_GUI.patch

```diff
@@ -0,0 +1,84 @@
+diff -ur nifti2dicom-0.4.11/CMakeLists.txt nifti2dicom-0.4.11-noGUI/CMakeLists.txt
```

Comment:
@crubb Please include a short description in the patch + mention author or upstream reference to where you got it.

Reply:
I hope it's Ok to not directly include my email address.

Reply:
Sure, just having some way of telling who the author is, that's fine :+1: